benchmarks/plot_benchmarks.py

```diff
@@ -28,31 +31,40 @@ def get_data_for_cluster(files, c):
     return clusters
 
 
-def get_all_data(cluster_range):
-    files = glob.glob("*.dat")
+def get_all_data(thread, cluster_range):
+    files = glob.glob(thread+"*.csv")
     data = {}
     for i in cluster_range:
         clusters = get_data_for_cluster(files, i)
         filenames = list(map(lambda x: x[0], clusters))
         ns = list(map(lambda x: x[1], clusters))
         df = get_data(filenames, ns)
+        if df:
+            df['n'] = [int(n) for n in df['n']]
         data[i] = df
     return data
 
 
-def plot_for_cluter(df):
-    # TODO: add option for data for multiple threads
+def plot_benchmark(df, label):
+    plt.errorbar(df['n'], df["mean"], yerr=df['std'], label=label)
 
-    plt.errorbar(df['n'], df["mean"], yerr=df['std'], label='Single thread')
-    # plt.yscale('log')
+
+def plot_for_cluster(single_thread, openmp, cluster):
```

Comment:
Should I combine no. of clusters in the figure, or plot separately??

Reply:
I think that the better option is to compare the time for each cluster separately. I'll leave it like this for now.